### PR TITLE
[zh-cn]sync kubelet-config-file configure-service-account

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/tasks/administer-cluster/kubelet-config-file.md
content/zh-cn/docs/tasks/configure-pod-container/configure-service-account.md

### CODE REVIEW

File: content/zh-cn/docs/tasks/configure-pod-container/configure-service-account.md [hunk 1]
```diff
@@ -195,12 +195,22 @@ The service account has to exist at the time the pod is created, or it will be r
...
 -->
 Pod 被创建时服务账号必须存在，否则会被拒绝。
 
 你不能更新已经创建好的 Pod 的服务账号。
 
+{{< note >}}
+`spec.serviceAccount` 字段是 `spec.serviceAccountName` 的已弃用别名。
+如果要从工作负载资源中删除字段，请在
+[Pod 模板](/zh-cn/docs/concepts/workloads/pods#pod-templates)上将这两个字段显式设置为空。
+{{< /note >}}
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/configure-pod-container/configure-service-account.md:212]:
```suggestion
{{< note >}}
<--
The `spec.serviceAccount` field is a deprecated alias for `spec.serviceAccountName`.
If you want to remove the fields from a workload resource, set both fields to empty explicitly
on the [pod template](/docs/concepts/workloads/pods#pod-templates).
-->
`spec.serviceAccount` 字段是 `spec.serviceAccountName` 的已弃用别名。
如果要从工作负载资源中移除这两个字段，请在
[Pod 模板](/zh-cn/docs/concepts/workloads/pods#pod-templates)上将这两个字段显式设置为空。
{{< /note >}}
```
It's a good choice to put both en and zh text into the note block.

File: content/zh-cn/docs/tasks/configure-pod-container/configure-service-account.md [hunk 2]
```diff
@@ -194,13 +194,25 @@ field of a pod to the name of the service account you wish to use.
...
 你不能更新已经创建好的 Pod 的服务账号。
 
+{{< note >}}
+<!--
+The `spec.serviceAccount` field is a deprecated alias for `spec.serviceAccountName`.
+If you want to remove the fields from a workload resource, set both fields to empty explicitly
+on the [pod template](/docs/concepts/workloads/pods#pod-templates).
+-->
+`spec.serviceAccount` 字段是 `spec.serviceAccountName` 的已弃用别名。
+如果要从工作负载资源中删除字段，请在
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configure-pod-container/configure-service-account.md:209]:
```suggestion
如果要从工作负载资源中删除这些字段，请在
```

**Comment by windsonsea** [comment]
Thanks.

/lgtm

**Pull request merged**